x-pack/packages/kbn-elastic-assistant/impl/assistant/assistant_overlay/index.tsx

```diff
@@ -38,9 +37,8 @@ export const UnifiedTimelineGlobalStyles = createGlobalStyle`
 
 export const AssistantOverlay = React.memo<Props>(({ currentUserAvatar }) => {
   const [isModalVisible, setIsModalVisible] = useState(false);
-  const [conversationTitle, setConversationTitle] = useState<string | undefined>(
-    WELCOME_CONVERSATION_TITLE
-  );
+  // Why is this named Title and not Id?
+  const [conversationTitle, setConversationTitle] = useState<string | undefined>(undefined);
```

Comment:
That's a a good question...probably leftover from before persistence when title was id? I would say go ahead and rename this, let's get as far away from conflating `id`/`title` as we can, as that's been problematic/messy as of late.